lbuild/format.py

```diff
@@ -9,47 +9,53 @@
 # governing this code.
 
 import os
-import re
 import sys
 import shutil
 import colorful
 
 import lbuild.node
 import lbuild.filter
 
-plain = not sys.stdout.isatty()
+PLAIN = not sys.stdout.isatty()
 WIDTH = shutil.get_terminal_size((100, 0)).columns
 
-color_scheme = {
+COLOR_SCHEMA = {
```

Comment:
```suggestion
COLOR_SCHEME = {
```
Color scheme as in a collection of colors.